sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -311,19 +311,19 @@ public virtual Response<Key> UpdateKey(KeyBase key, IEnumerable<KeyOperation> ke
         /// Vault. Note: The cryptographic material of a key itself cannot be changed.
         /// This operation requires the keys/update permission.
         /// </remarks>
-        /// <param name="key">The <see cref="KeyBase"/> object with updated properties.</param>
+        /// <param name="key">The <see cref="KeyProperties"/> object with updated properties.</param>
         /// <param name="keyOperations">Optional list of supported <see cref="KeyOperation"/>. If null, no changes will be made to existing key operations.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="key"/> or <paramref name="keyOperations"/> is null, or <see cref="KeyBase.Version"/> of <paramref name="key"/> is null.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="key"/> or <paramref name="keyOperations"/> is null, or <see cref="KeyProperties.Version"/> of <paramref name="key"/> is null.</exception>
         /// <exception cref="RequestFailedException">The server returned an error.</exception>
-        public virtual async Task<Response<Key>> UpdateKeyAsync(KeyBase key, IEnumerable<KeyOperation> keyOperations = null, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<Key>> UpdateKeyPropertiesAsync(KeyProperties key, IEnumerable<KeyOperation> keyOperations = null, CancellationToken cancellationToken = default)
```

Comment:
>UpdateKeyPropertiesAsync [](start = 49, length = 24)

In SecretClient the analogous method is UpdateProperties vs. UpdateSecretProperties.  I expect that this inconsistency is the same throughout the client we should create an issue to track unifying this to one pattern of the other.

Reply:
It is inconsistent across the libraries, yes. I noticed this while working on this change as well, but figured it was the agreed-upon design. I'll open a tracking bug, though.